ui/core.js

```diff
@@ -49,6 +49,29 @@ $.extend( $.ui, {
 		SPACE: 32,
 		TAB: 9,
 		UP: 38
+	},
+
+	safeActiveElement: function( document ) {
+		var activeElement;
+
+		// Support: IE 9 only
+		// IE9 throws an "Unspecified error" accessing document.activeElement from an <iframe>
+		try {
+			activeElement = document.activeElement;
+		} catch ( error ) {
+			activeElement = document.body;
+		}
+
+		return activeElement;
+	},
+
+	safeBlur: function( element ) {
+
+		// Support: IE9 - 10 only
+		// If the <body> is blurred, IE will switch windows, see #9420
+		if ( element && element.nodeName.toLowerCase() !== "body" ) {
```

Comment:
I believe the reason for that is that if you ever pass ```event.target``` or the context of an event handler you may end up passing in the window. I don't believe ```document.activeElement``` can ever be set to the window. I guess the basic assumption was that ```$( window ).blur()``` is unsafe in the same way as ```$( "body" ).blur()``` in that it results in the window losing focus. However, I've now tested ```$( window ).blur()``` as far back as IE6 and it does not seem to do anything, so I guess attempting to blur the window is safe.

I used the following test:

```JS
<!doctype html>
<html>
<head>
	<script src="http://code.jquery.com/jquery-1.11.1.js"></script>
	<script>
setTimeout( function() {
	$( "#console" ).text( "blurring window\n" );
	$( window ).blur();
}, 5000 );
	</script>
</head>
<body>
	<pre><code id="console"></code></pre>
	<input>
</body>
</html>
```

and I tried both with focus initially set to default and with focus immediately set on the input by my clicking on it.